### PR TITLE
Bones now are reasonably sized when on the ground but still display full detail when interacted with.

### PR DESCRIPTION
**Pull request by Hopekz**
Okay, Halloween is over. Time to fix these big bones. 
These big bones look pretty good though so I can only think of one way to preserve this full detail.

Using this simple BYOND trick bones are now reasonably sized when on the ground but display full 32x32 detail when interacted with.

aOH42NhD3e

## Changelog
:cl: Hopek
add: Bones now are reasonably sized when on the ground but still display full detail when interacted with.
/:cl:

### CODE REVIEW

**Review by TorinoFermic — APPROVED** [review]
I like the pick up effect when taken

**Comment by Hopekz** [comment]
SpooOOoky
"image

**Review by genessee596 — APPROVED** [review]
age of giants is over

**Comment by genessee596** [comment]
has maintainer and delegate approval, very nice

**Pull request merged**
